jumpscale/install/Dockerfile

```diff
@@ -22,6 +22,7 @@ RUN if [ "$TRC" = "true" ] ; then wget https://github.com/threefoldtech/tcproute
     && chmod +x /sbin/trc ; fi
 
 COPY rootfs /
+COPY certbot/certbot /etc/cron.d
```

Comment:
can be renamed to certbot/cronjob?

Reply:
Your idea is better to make it clear in the source folder, but this name will not be clear in the destination folder, right?

Reply:
we can name the destination file as needed

Reply:
Ok, I will modify it in the next commit.